org.civicrm.reqangular/test/services/file-upload_test.js

```diff
@@ -90,6 +89,32 @@ define([
           expect(firstObject.file).toBeDefined();
         });
       });
+
+      describe('onBeforeUploadItem()', function () {
+        beforeEach(function () {
+          spyOn(uploader, 'onBeforeUploadItem').and.callThrough();
+          uploader.onBeforeUploadItem({ formData: [] });
+        });
+
+        it('gets called', function () {
+          //verify if method is getting called and hence a simple case
+          expect(uploader.onBeforeUploadItem).toHaveBeenCalled();
```

Comment:
Again, what's the point of this? You are calling this method yourself, how can this test ever fail?